packages/mask/src/plugins/EVM/constants.ts

```diff
@@ -175,48 +176,69 @@ export const PLUGIN_PROVIDERS: Web3Plugin.ProviderDescriptor[] = [
         type: ProviderType.MaskWallet,
         name: 'Mask Network',
         icon: new URL('./assets/maskwallet.png', import.meta.url),
+        enableRequirements: {
+            supportedChainIds: getEnumAsArray(ChainId).map((x) => x.value),
```

Comment:
I think explicitly listing all chain IDs can prevent accidental fails in the future.